app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,76 @@
+class AdminController < ApplicationController
```

Comment:
The AdminController should be the base one in the adminstration.

Then we should have other controllers for each section which inherit AdminController:

For example:

- InstancesController < AdminController
- BillingController < AdminController
- AccountController < AdminController
- SupportController < AdminController

I can do it if you want - just let me know

Reply:
Adding admin scope, I think it's going to be cleaner.